WinUIGallery/Controls/ControlExample.xaml

```diff
@@ -138,26 +160,38 @@
                 <muxc:Expander.Header>
                     <TextBlock Text="Source code" />
                 </muxc:Expander.Header>
+                <Grid RowSpacing="8">
```

Comment:
Maybe this could be 16, to provide a bit more whitespace?

Reply:
done